addons/mail/static/src/core/common/composer.xml

```diff
@@ -174,7 +174,7 @@
 
 <t t-name="mail.Composer.dropdownActions">
     <t t-foreach="partitionedActions.other" t-as="action" t-key="action.id">
-        <DropdownItem class="'btn rounded-0 d-flex align-items-center px-2 py-2 m-0'" onSelected="(ev) => action.onClick?.(ev)" attrs="{ 'name': action.id, 'data-hotkey': action.hotkey, 'disabled': action.disabledCondition or areAllActionsDisabled }">
+        <DropdownItem t-if="!action.disabledCondition" class="'btn rounded-0 d-flex align-items-center px-2 py-2 m-0'" onSelected="(ev) => action.onClick?.(ev)" attrs="{ 'name': action.id, 'data-hotkey': action.hotkey }">
```

Comment:
`disabledCondition` determines whether the button should be disabled or not. This is not the same as whether the button is present or not, which is determined by `condition()` (and pre-filtered in `partitionedActions`).

So the `attrs={ 'disabled` }` must likely be kept.

Reply:
DropdownItem generates `<span>` element, disabled does not affect them.